CTP/common/ext/run_shell_git.sh

```diff
@@ -0,0 +1,134 @@
+#     the documentation and/or other materials provided with the distribution.
+# 
+#   * Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products 
+#     derived from this software without specific prior written permission.
+# 
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, 
+# INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE 
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, 
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR 
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE 
+# USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE. 
+#
+
+#init conf for shell
+url=""
+role=""
+branch=""
+category=""
+db_charset=""
+testcase_root=""
+testcase_timeout=""
+excluded_list=""
+fail_case_retry_count=""
+feedback_type="database"
+tool_home=""
+
+git_user=""
+git_pwd=""
+git_email=""
+
+svn_user=""
+svn_pwd=""
+cubrid_common_url=""
+
+coverage_controller_ip=""
+coverage_controller_user=""
+coverage_controller_pwd=""
+coverage_controller_port=""
+coverage_controller_target_dir=""
+coverage_collaborate_url=""
+
+ctp_test_conf=${CTP_HOME}/conf/shell.conf
+ctp_common_conf=${CTP_HOME}/conf/dailyqa.conf
+
+git_user=`ini.sh $ctp_common_conf "git.user"`
+git_pwd=`ini.sh $ctp_common_conf "git.pwd"`
+git_email=`ini.sh $ctp_common_conf "git.email"`
+
+svn_user=`ini.sh $ctp_common_conf "svn.user"`
+svn_pwd=`ini.sh $ctp_common_conf "svn.pwd"`
+cubrid_common_url=`ini.sh $ctp_common_conf "cubrid.commom.url"`
+
+url=$url
+coverage_controller_ip=$MKEY_COVERAGE_UPLOAD_IP
+coverage_controller_user=$MKEY_COVERAGE_UPLOAD_USER
+coverage_controller_pwd=$MKEY_COVERAGE_UPLOAD_PWD
+coverage_controller_port=$MKEY_COVERAGE_UPLOAD_PORT
+testcase_timeout=`getMsgValue $MKEY_TESTCASE_TIMEOUT 7200`
+fail_case_retry_count=`getMsgValue $MKEY_MAX_RETRY_COUNT 0`
+db_charset=`getMsgValue $MKEY_TESTING_DEFAULT_CHARSET en_US`
+
+if [  "$BUILD_TYPE" == "coverage" ];then
+        role="--role-coverage"
+        coverage_controller_target_dir=${MKEY_COVERAGE_UPLOAD_DIR}/${BUILD_ID}/new
+        feedback_type="file"
+	coverage_collaborate_url=$src_url
+fi
+
+
+if [ $BUILD_IS_FROM_GIT -eq 1 ];then
+	url=$url
+	branch=$BUILD_SCENARIO_BRANCH_GIT
+	category=$BUILD_SCENARIOS
+	testcase_root=cubrid-testcases-private-ex/shell	
+	excluded_list=cubrid-testcases-private-ex/shell/config/daily_regression_test_excluded_list_linux.conf	
+	tool_home=${CTP_HOME}
+
+	#update configuration file
+	ini.sh -u "main.git.user=$git_user" $ctp_test_conf
```

Comment:
It's better not to use below parameters in shell configuration.
main.git.user
main.git.pwd
main.git.email.

Let's provide better solution.